wpimath/src/main/java/edu/wpi/first/wpiutil/math/Matrix.java

```diff
@@ -22,8 +21,7 @@
 import edu.wpi.first.math.WPIMathJNI;
 import edu.wpi.first.wpiutil.math.numbers.N1;
 
-/**
- * A shape-safe wrapper over Efficient Java Matrix Library (EJML) matrices.
+/*
  *
```

Comment:
Something weird happened here. Shouldn't there be two asterisks?

Reply:
This actually should be removed. Not sure why the doc comment's gone.